src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -346,6 +347,23 @@ public boolean canAdministerProject(String projectID, String organizationId) {
         return false;
     }
 
+    /** Check that user can administer project. Organization ID is drawn from persisted project. */
+    public boolean canAdministerProject(String projectId) {
+        if (canAdministerApplication()) return true;
+        com.conveyal.datatools.manager.models.Project p = Persistence.projects.getById(projectId);
```

Comment:
Is there a way to not have the full classpath here?

Reply:
Not that I'm aware of.